### PR TITLE
add VCS and Xcelium wrappers that apply wake resource variables

### PR DESCRIPTION
**Pull request by albertchen-sifive**
https://github.com/sifive/api-generator-sifive/pull/14 for VCS and Xcelium

### CODE REVIEW

File: sim/vcs.wake [hunk 1]
```diff
@@ -53,12 +57,12 @@ def compileInputs options =
...
 
-  mkdir tmpDir, mkdir outputDir, includeSources ++ srcFiles
+  source vcsWrapperScript, mkdir tmpDir, mkdir outputDir, includeSources ++ srcFiles
 
 global def doVCSCompile options =
   def outputDir = options.getVCSCompilePlanOutputDir
-  def cmdline = "vcs", options.compileCmdlineArgs
-  def inputs = options.compileInputs
+  def cmdline = options.compileCmdlineArgs
+  def inputs = source vcsWrapperScript, options.compileInputs
```
Comment (richardxia) [inline sim/vcs.wake]: Am I reading this wrong, or are you adding the `vcs_wrapper` script twice? I see it both here and on line 60, in the `def compileInputs` function.

File: sim/xcelium.wake [hunk 2]
```diff
@@ -52,13 +55,12 @@ def compileInputs options =
...
-  mkdir outputDir, includeSources ++ srcFiles ++ homeSources# ++ includeDirs
+  source xrunWrapperScript, mkdir outputDir, includeSources ++ srcFiles ++ homeSources# ++ includeDirs
 
 global def doXceliumCompile options =
   def outputDir = options.getXceliumCompilePlanOutputDir
-  def cmdline = "xrun", options.compileCmdlineArgs
-
-  def inputs = options.compileInputs
+  def cmdline = options.compileCmdlineArgs
+  def inputs = source xrunWrapperScript, options.compileInputs
```
Comment (richardxia) [inline sim/xcelium.wake]: Same question here.

**Review by richardxia — APPROVED** [review]
LGTM, modulo question about the potentially doubly-included wrapper scripts.

File: sim/vcs.wake (hunk 1 above)
Comment (albertchen-sifive) [reply sim/vcs.wake]: you're right this is redundant. fixed

**Pull request merged**